Source/Scene/Expression.js

```diff
@@ -62,7 +62,16 @@ define([
         asin : Math.asin,
         atan : Math.atan,
         radians : CesiumMath.toRadians,
-        degrees : CesiumMath.toDegrees
+        degrees : CesiumMath.toDegrees,
+        sign : Math.sign,
+        floor : Math.floor,
+        ceil : Math.ceil,
+        round : Math.round,
+        exp : Math.exp,
+        exp2 : CesiumMath.exp2,
+        log : Math.log,
+        log2 : Math.log2,
+        fract : CesiumMath.fract
```

Comment:
>I've done the basic, where fract(x) when x < 0 outputs x - floor(x). This means that the output of fract(-2.25) would be 0.75. If there's another way you would prefer this to be handled, please let me know.

This is the way GLSL does it, so that's good.